src/Core/Utility/Form.php

```diff
@@ -62,7 +62,7 @@ protected static function jsonControl($value) : array
     {
         return [
             'type' => 'textarea',
-            'v-jsoneditor' => '',
+            'v-jsoneditor' => 'true',
```

Comment:
why this change here?

Reply:
because i need a truthy value in the input-dynamic-attribute vue component